### PR TITLE
feat(LEMS-2720): add aria label to angle graph

### PR DESCRIPTION
**Pull request by anakaren-rojas**
## Summary:
Adds aria label to angle points

Issue: LEMS-2720

## Test plan:
1. Navigate to Editor Page in [Chromatic](https://650db21c3f5d1b2f13c02952-tsstzipopb.chromatic.com/?path=/docs/perseuseditor-editorpage--docs)
2. Add an interactive graph widget
3. With a screen reader running, select a point on the angle 
4. Tick the `show angle measures` box 
5. Ticket the `allow reflex angles` box

### Expected functionality 
1. Aria label for the terminal and initial side should follow the format: 
`Point #, {sideName} side at {x} comma {y}`
or for vertex when `show angle measures` isn't ticked
`Point #, vertex at {x} comma {y}`

2. Aria label for the vertex when `show angle measures` is ticked should follow the format:
`Point #, vertex at {x} comma {y}. Angle {angleMeasure} degrees`

3. When `show reflex angles` is ticked, reflex angle measure should be reflected in the vertex aria label 

## Screen Recording
https://github.com/user-attachments/assets/88a55c4a-a2f7-4e5b-a724-05194e70ddc9

### CODE REVIEW

**Comment by nishasy** [comment]
Can you add to your PR description the things that will be coming later? I don't want to make comments about stuff if you're already aware of it and will be doing it in a future PR.

**Comment by anakaren-rojas** [comment]
> Can you add to your PR description the things that will be coming later? I don't want to make comments about stuff if you're already aware of it and will be doing it in a future PR.

For sure @nishasy 
This is what I added: 
- Add aria label and description to whole angle graph
- Add notifications when a point is moved on the graph

File: packages/perseus/src/widgets/interactive-graphs/graphs/angle.tsx [hunk 1]
```diff
@@ -95,6 +97,64 @@ function AngleGraph(props: AngleGraphProps) {
         showAngles: showAngles || false, // Whether to show the angle or not
     };
 
+    const angleMeasure = getClockwiseAngle(
+        [endPoints[0], centerPoint, endPoints[1]],
+        allowReflexAngles,
+    );
```
Comment (nishasy) [inline packages/perseus/src/widgets/interactive-graphs/graphs/angle.tsx:103]: I think you might have to trim this angle measure so it's only integers or maybe only shows 1 or 2 significant figures. I think you can use `srFormatNumber()` for this here too.

image

**Comment by catandthemachines** [comment]
Question, can we truncate the angle radius degrees? Or did Caitlyn state that we wanted them that long?
image

File: packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx [hunk 2]
```diff
@@ -481,11 +481,9 @@ describe("MafsGraph", () => {
             />,
         );
 
-        // The middle coords are actually the first point because we want
-        // the vertex to show up first in the point tab order.
-        expectLabelInDoc("Point 1 at 0 comma 0");
-        expectLabelInDoc("Point 2 at -1 comma 1");
-        expectLabelInDoc("Point 3 at 1 comma 1");
+        expectLabelInDoc("Point 1, terminal side at -1 comma 1");
```
Comment (catandthemachines) [inline packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx:484]: Can we add some tests here for the aria-label for when it reads out the vertex angle? :)

**Comment by anakaren-rojas** [comment]
> Question, can we truncate the angle radius degrees? Or did Caitlyn state that we wanted them that long? image

She definitely didn't ask for them to be that long 😢. What's interesting about this is, until about an hour ago, this wasn't happening and now I can't get it to _stop_ happening @catandthemachines

**Review by catandthemachines — APPROVED** [review]
Approved! Just would like one more test and a question on the length of the angle value.

File: packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx [hunk 3]
```diff
@@ -481,11 +481,9 @@ describe("MafsGraph", () => {
             />,
```
Comment (nishasy) [inline packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx:481]: Something I should have done a long time ago - can you put a comment above line 471 saying "[0, 0] is the vertex"

**Review by nishasy — APPROVED** [review]
The strings all look correct to me. My one suggestion is to trim the string for the angle measure.

Looking forward to seeing this come together!

**Comment by catandthemachines** [comment]
> > Question, can we truncate the angle radius degrees? Or did Caitlyn state that we wanted them that long? image
> 
> She definitely didn't ask for them to be that long 😢. What's interesting about this is, until about an hour ago, this wasn't happening and now I can't get it to _stop_ happening @catandthemachines

I think a simple Math.round() can solve this issue. But unless stated in Caitlyns SR Documentation, we should ask her what we can round to.

**Pull request merged**